rust/arrow/src/compute/kernels/filter.rs

```diff
@@ -112,7 +112,7 @@ impl<'a> CopyNullBit for NullBitSetter<'a> {
     }
 
     fn null_buffer(&mut self) -> Buffer {
-        self.target_buffer.resize(self.target_index).unwrap();
+        self.target_buffer.resize(self.target_index);
```

Comment:
These are most probably generating clippy warnings.

Reply:
You are likely correct -- we should eventually turn on a clippy for the CI checks in this crate